packages/replica-healthcheck/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@eth-optimism/replica-healthcheck",
+  "version": "0.1.0",
+  "private": true,
+  "main": "dist/index",
+  "files": [
+    "dist/index"
```

Comment:
Are you sure this is the only file to be distributed?

Reply:
This is only useful for npm packages if I recall